src/seraphis_main/contextual_enote_record_types.h

```diff
@@ -237,14 +271,25 @@ rct::xmr_amount amount_ref(const SpContextualEnoteRecordV1 &record);
 //////////////////////////////////////////////// Joint /////////////////////////////////////////////////////
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
+// QUESTION: Is this the way? (creating a new variant for return value)
```

Comment:
I need to give it a deeper pass, but it seems like a solid approach to me so far